hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/ListSnapshotDiffHandler.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.hadoop.ozone.shell.snapshot;
+
+import org.apache.hadoop.ozone.client.OzoneClient;
+import org.apache.hadoop.ozone.om.helpers.SnapshotDiffJob;
+import org.apache.hadoop.ozone.shell.Handler;
+import org.apache.hadoop.ozone.shell.OzoneAddress;
+import org.apache.hadoop.ozone.shell.bucket.BucketUri;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * ozone sh snapshot listSnapshotDiff.
+ */
+@CommandLine.Command(name = "listSnapshotDiff",
```

Comment:
Similar to [HDDS-8789](https://issues.apache.org/jira/browse/HDDS-8789), I was thinking if we should simplify it and just call it listdiffJob. The command would be `ozone sh snapshot listdiffJob`. It would be great if we can do something like `ozone sh snapshot list diffJob` or `ozone sh snapshot ls diffJob`

Reply:
We can't have spaces, because the second word is considered part of the BucketURI. Another way to go about this, would be to make the command part of the diff command and have something like `ozone sh snapshot diff /vol/bucket list` or `ozone sh snapshot diff /vol/bucket ls`. The second approach might end up a bit complicated.

Reply:
@hemantk-12 That's how it looks.

```
bash-4.2$ ozone sh snapshot
Incomplete command
Usage: ozone sh snapshot [-hV] [COMMAND]
Snapshot specific operations
  -h, --help      Show this help message and exit.
  -V, --version   Print version information and exit.
Commands:
  create                                    Create a snapshot
  delete                                    Delete a snapshot
  list, ls                                  List snapshots for the buckets.
  snapshotDiff                              Get the differences between two
                                              snapshots
  listDiff, listDiffJob, lsDiff, lsDiffJob  List snapshotDiff jobs for a bucket.
```